metaflow/plugins/events_decorator.py

```diff
@@ -178,7 +231,7 @@ def flow_init(
             raise MetaflowException("No event(s) specified in *@trigger* decorator.")
 
         # same event shouldn't occur more than once
-        names = [x["name"] for x in self.triggers]
+        names = [x["name"] for x in self.triggers if not isinstance(x, DeployTimeField)]
```

Comment:
not fully complete test, should be something like x not instance of deploytimefield but also x["name"] not a deploy time field.